datasets/mx/deputies/crawler.py

```diff
@@ -0,0 +1,66 @@
+import re
+import requests
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+def crawl_item(input_dict: dict, context: Context):
+
+    entity = context.make("Person")
+    entity.id = context.make_id(input_dict["NombreCompleto"])
+
+    entity.add("lastName", input_dict.pop("Nombre"))
+    entity.add("firstName", input_dict.pop("PrimerApellido"))
+    entity.add("name", input_dict.pop("NombreCompleto"))
+
+    if input_dict["Telefono"] is not None:
+        entity.add("phone", "+52"+input_dict.pop("Telefono"))
+
+    entity.add("email", input_dict.pop("Correo"))
+
+    position = h.make_position(context, "Member of the Chamber of Deputies", country="mx")
+    categorisation = categorise(context, position, is_pep=True)
+
+    occupancy = h.make_occupancy(
+            context,
+            entity,
+            position,
+            True,
+            categorisation=categorisation,
+        )
+
+    if occupancy is None:
+        return
+
+    context.emit(entity, target=True)
+    context.emit(position)
+    context.emit(occupancy)
```

Comment:
```suggestion
    context.emit(occupancy)
    context.audit_data(input_dict)
```

adding audit_data shows a few more fields we can use:
- SegundoApellido 
- Partido -> Person:politicial

the rest you can add to the audit ignore argument

Reply:
I added the Partido to `political` and `SegundoApellido` in the `lastName`